Source/MLX/MLXArray.swift

```diff
@@ -151,6 +153,11 @@ public final class MLXArray {
         #endif
         case is Float32.Type: return mlx_array_item_float32(array_ctx) as! T
         case is Float.Type: return mlx_array_item_float32(array_ctx) as! T
+        case is Complex<Float32>.Type:
+            // mlx_array_item_complex64() isn't visible in swift so read the array
+            // contents
+            let ptr = UnsafePointer<Complex<Float32>>(mlx_array_data_complex64(ctx))!
+            return ptr.pointee as! T
```

Comment:
`item()` readout of `Complex<Float32>`.  There isn't an equivalent `bfloat16` readout because there isn't a type for it.  You can read it as `Float32` or `Float16` and it will convert on the fly.